stack.yaml

```diff
@@ -2,13 +2,13 @@
 # For more information, see: http://docs.haskellstack.org/en/stable/yaml_configuration.html
 
 # Specifies the GHC version and set of packages available (e.g., lts-3.5, nightly-2015-09-21, ghc-7.10.2)
-resolver: lts-5.1
+resolver: lts-5.9
 
 # Local packages, usually specified by relative directory name
 packages:
 - '.'
 # Packages to be pulled from upstream that are not in the resolver (e.g., acme-missiles-0.3)
-extra-deps: []
+extra-deps: [aeson-0.11.0.0]
```

Comment:
Mkay but if you have a dependency conflict then it's not an issue of the masterserver. Stack surely supports some notation for having a minimal version of some package, setting it to "we require exactly 0.11.0.0" is really not good.